HelikarLab/candis
# Arrange compartments in ToolBox Component as desired.

Solves #22 #23 , resolves import error names (on linux OS, import names are case sensitive), added `yarn-error.log` in gitignore for yarn users. 

###### Screenshots/GIFs:

  - Screenshot
 
![screenshot from 2018-04-15 00-28-32](https://user-images.githubusercontent.com/20624380/38771752-1432fc38-4046-11e8-9f7b-088cf0c8bbb4.png)

###### Steps To Test:

  - Steps
Since I have already included the bundle.min.js file, one have to fork this changes, clone locally and then run `python -m candis` from root directory to see the changes.

###### TODOs:

  - None
